examples/appliance_proxy_configuration.py

```diff
@@ -50,10 +50,10 @@
 
 # Get proxy configuration from appliance
 print("\nGet proxy configuration from appliance:\n ")
-proxy_info = proxy.get_all()
```

Comment:
we still have get_all() in [library file](https://github.com/HewlettPackard/oneview-python/blob/f1856b68b0b74d3d1b75da2b47e193b601f9fb5a/hpeOneView/resources/settings/appliance_proxy_configuration.py#L39). 
Either remove it from their if not supported or set a separate example.

Reply:
get_all is being used in get_by_proxy method.. so no need to set separate example.